python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -979,6 +980,83 @@ def convert_to_cudf(cp_arrays):
 
         return ddf
 
+    def decompress_to_edgelist(
+        self,
+        original: bool = True
```

Comment:
I think this would be more descriptive if it was named `return_unrenumbered_edgelist`